boards/cy8cproto_62_4343_w/.cargo/config.toml

```
@@ -0,0 +1,14 @@
+# Licensed under the Apache License, Version 2.0 or the MIT License.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+# Copyright OxidOS Automotive SRL 2025.
+
+include = [
+	"../../cargo/tock_flags.toml",
+	"../../cargo/unstable_flags.toml",
```

```suggestion
  "../../cargo/tock_flags.toml",
  "../../cargo/unstable_flags.toml",
```

Just a nit, we try our best to ensure we have no tab characters in the Tock source code (as that makes for inconsistent rendering with different editors). It seems like we should include these files in the automatic checks as well.